mule-extensions-api/src/main/java/org/mule/runtime/extension/api/loader/DeclarationEnricherPhase.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.extension.api.loader;
+
+import org.mule.runtime.api.meta.model.ModelProperty;
+import org.mule.runtime.api.meta.model.declaration.fluent.ExtensionDeclarer;
+
+import java.util.Collection;
+
+
+/**
+ * This {@link DeclarationEnricherPhase phases} are used to create a {@link DeclarationEnricher}s execution chain,
+ * each {@link DeclarationEnricherPhase} describes a particular enrichment state from which {@link DeclarationEnricher}s can
```

Comment:
be more explicit regarding the fact that the order in which each enricher is to be applied depends on its phase